docs/decisions/arch-diagrams/refund-flow.mmd

```diff
@@ -0,0 +1,39 @@
+sequenceDiagram
```

Comment:
Suggestion: I believe it is better to separate out iOS and Android refund flows in their own diagrams because the flow is different for both.